juntagrico/util/management.py

```diff
@@ -1,28 +1,25 @@
 import hashlib
+import itertools
 import random
 import string
 
 from juntagrico.models import Share, Subscription, TSST, TFSST
-from juntagrico.dao.subscriptiontypedao import SubscriptionTypeDao
 from juntagrico.mailer import send_welcome_mail
 from juntagrico.mailer import send_been_added_to_subscription
 from juntagrico.mailer import send_share_created_mail
 
 
 def password_generator(size=8, chars=string.ascii_uppercase + string.digits):
     return ''.join(
-        random.choice(chars) for x in range(size))
+        random.choice(chars) for _ in range(size))
 
 
 def create_subscription(start_date, depot, selected_subscriptions):
     subscription = Subscription()
     subscription.start_date = start_date
     subscription.depot = depot
     subscription.save()
-    for sub_type, amount in selected_subscriptions.items():
-        for i in range(amount):
-            TSST.objects.create(type=sub_type, subscription=subscription)
-            TFSST.objects.create(type=sub_type, subscription=subscription)
+    replace_subscription_types(subscription, selected_subscriptions)
```

Comment:
shouldnt _obj here be set to TSST

Reply:
no, future subs must be replaced as well. _obj should not be used externally.